app/src/main/java/edu/grinnell/kdic/RadioService.java

```diff
@@ -109,12 +88,11 @@ public int onStartCommand(Intent intent, int flags, int startId) {
 
         if (intent != null && intent.getAction() != null) {
```

Comment:
Let's keep `onCreate(..)` short. Extract all of this into some other method and call it in `onCreate(...)`